src/main/java/org/olf/folio/order/OrderImport.java

```diff
@@ -105,12 +112,20 @@ public  JSONArray  upload(String fileName) throws IOException, InterruptedExcept
 			 lookupUtil.setApiService(apiService);
 			 lookupUtil.load();
 			 this.lookupTable = lookupUtil.getReferenceValues(token);
+			 String billingEndpoint = baseOkapEndpoint+"configurations/entries?limit=100";
```

Comment:
What do you think about dropping the limit and instead using the [`query` URL param](https://s3.amazonaws.com/foliodocs/api/mod-configuration/p/config.html#configurations_entries_get) to only return `tenant.addresses`?

```suggestion
			 String billingEndpoint = baseOkapEndpoint+"configurations/query=(configName==tenant.addresses)";
```

Not only does it simplify the logic for `lookupUtil.getBillingAddresses()`, but it also avoids potentially missing addresses as the number of configuration entries grow beyond some arbitrary limit.

> Configuration entries count currently `56` for training and `58` for test

Reply:
The URL you suggested won't work.  This one will:
String endpoint = this.getBaseOkapEndpoint()+"configurations/entries?query=(configName==tenant.addresses)";

Reply:
Sorry for my typo